### ISSUE TITLE
tag #? is not a descendant of <RCTShadowView

### ISSUE THREAD

**Issue by carlosgarbiatti** [APSL/react-native-keyboard-aware-scroll-view#61]
Hi all, firstly, thanks for the component and for your time of maintaining it. 

I'm using native-base which uses react-native-keyboard-aware-scroll-view. I was aware of the issue #25 and was waiting for the new react-native and  react-native-keyboard-aware-scroll-view versions (respectively 0.32.0 and 0.2.0). Even after the upgrade, I'm still having this issue on IOS when I focus an input and keyboard shows up. 

![image](https://cloud.githubusercontent.com/assets/5911810/18036360/93b3d1a2-6dbc-11e6-831b-c4f81bab3487.png)

My package.json looks like this:

```
"dependencies": {
    "buffer": "^4.9.0",
    "native-base": "0.5.7",
    "react": "~15.3.0",
    "react-native": "0.32.0",
    "react-native-code-push": "^1.13.5-beta",
    "react-native-fbsdk": "^0.3.0",
    "react-native-i18n": "0.0.8",
    "react-native-image-picker": "^0.21.5",
    "react-native-keyboard-aware-scroll-view": "0.2.0",
    "react-native-multiple-choice": "0.0.8",
    "react-native-swiper": "^1.4.9",
    "react-native-vector-icons": "^2.0.3",
    "react-redux": "^4.4.5",
    "redux": "^3.5.2",
    "redux-logger": "^2.6.1",
    "redux-persist": "^3.2.2",
    "redux-promise": "^0.5.3",
    "redux-thunk": "^2.1.0",
    "remote-redux-devtools": "^0.3.4",
    "rx": "^4.1.0"
  }
```

Am I missing something or is this a bug?

Thanks!

**Comment by alvaromb** [comment]
Hello @carlosgarbiatti and thanks for the report!

Can you double check that you're running `0.32.0`? I'm currenty running an iPad project with that version and with `0.2.0` of the keyboard component and I don't get these errors.

I'll remove the `node_modules/` folder and reinstall to check again if we have any problem. You can try the same and test if it works after running `npm i` again.

**Comment by sankhadeeproy007** [comment]
Hi @carlosgarbiatti! I am an author of `native-base` and I can confirm that it works fine with RN 0.32.0 and 0.2.0. Deleting the `node_modules` directory as suggested should do the trick. 
Btw @alvaromb thanks for this lovely library. You saved me a lot of headache!

**Comment by alvaromb** [comment]
Thanks @sankhadeeproy007!

Closing the issue.

**Comment by carlosgarbiatti** [comment]
Hey guys thanks for the support. 

Even after deleting `node_modules`, I still get the error 😕. I tried two different versions of native_base (0.5.4 and the latest 0.5.7). I double checked and the package.json in both `react-native` and `react-native-keyboard-aware-scroll-view` and the versions seems to be correct. 

@sankhadeeproy007, can you please post your package.json so I can check the dependencies I'm using match yours.

Thanks again!

**Comment by alvaromb** [comment]
Hey @carlosgarbiatti!

I would need some source code to test why it isn't working. This would happen when you're try to scroll to a `TextInput` that it's not present in the current `ScrollView`.

**Comment by carlosgarbiatti** [comment]
Sorry guys, I've found the issue! I'm quite new to Javascript world, so sorry to bother you guys with stupid mistakes. 

The problem was that `native-base` (0.5.4) brings an old version of `react-native-keyboard-aware-scroll-view` as its transitive dependency. I deleted `node_modules/native-base/node_modules/react-native-keyboard-aware-scroll-view` folder and it works fine!

I just need to figure out now how to exclude this transitive dependency 😅 .

Thanks again!
